test/cql-pytest/test_udf.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+#############################################################################
+# Tests for user-defined functions (UDF)
+#############################################################################
+
+import pytest
+from util import unique_name, new_function
+from cassandra.protocol import InvalidRequest
+
+# Unfortunately, while ScyllaDB and and Cassandra support the same UDF
+# feature, each supports a different choice of languages. In particular,
+# Cassandra supports Java and ScyllaDB doesn't - but ScyllaDB supports
+# Lua. The following fixture can be used to check if the server supports
+# Java UDFs - and if it doesn't, the test should use Lua.
+@pytest.fixture(scope="session")
+def has_java_udf(cql, test_keyspace):
+    try:
+        with new_function(cql, test_keyspace, "(i int) CALLED ON NULL INPUT RETURNS int LANGUAGE java AS 'return 42;'"):
+            return True
+    except InvalidRequest as e:
+        # When the language "java" is not supported, Scylla prints the
+        # error message "Language 'java' is not supported". Cassandra
+        # isn't expected to fail here because it does support Java.
+        # Any other error here indicates an unexpected bug.
+        assert "Language 'java' is not supported" in str(e)
+        return False
+
+# Test that if we have two functions with the same name, just with
+# different parameters, DROP FUNCTION requires disambiguating which
```

Comment:
> two functions with the same name, just with different parameters

I see you're avoiding the shorter `overloaded functions` wording. AFAIK overloaded functions are common in cpp & java. That's OK, I guess. No action point here, just a comment.

Reply:
I think I used it somewhere (commit message?). I'll use it here too.